data_generator/erc20-gateways/deposit_proof.js

```diff
@@ -0,0 +1,154 @@
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const BN = require('bn.js');
+const rlp = require('rlp');
+const fs = require('fs');
+
+const ERC20Gateway = artifacts.require('ERC20Gateway');
+
+const Web3 = require('web3');
+const { AccountProvider } = require('../../test/test_lib/utils');
+const Utils = require('../../test/test_lib/utils');
+// const web3 = require('../../test/test_lib/web3');
```

Comment:
```suggestion
```

Reply:
File deleted.
This can only be tested in an integration test.
The channel identifier will change with each deployment, so the same proof data cannot be used.